### PR TITLE
Change Live Share docs for LiveState & LiveCanvas

### PR DESCRIPTION
**Pull request by ryanbliss**
- Updated `LiveState` docs to reflect latest interface changes
- Updated `LiveCanvas` docs to reflect new features for stroke import/export

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-live-share-canvas.md](https://github.com/MicrosoftDocs/msteams-docs/blob/80c14a1da4ade99193657f3d71608a3ed8fe6768/msteams-platform%2Fapps-in-teams-meetings%2Fteams-live-share-canvas.md) | 93 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a6a7cafb-518e-4013-99c4-13739f06a2b6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/teams-live-share-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/80c14a1da4ade99193657f3d71608a3ed8fe6768/msteams-platform%2Fapps-in-teams-meetings%2Fteams-live-share-capabilities.md) | 93 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6c218f55-2565-4c34-8987-b96868a426ce) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/apps-in-teams-meetings/teams-live-share-canvas.md [hunk 1]
```diff
@@ -334,6 +334,28 @@ document.getElementById("line-tip-size").onclick = () => {
...
+const strokes = inkingManager.exportRaw();
+
+// Optionally clear out existing strokes, and import strokes
+inkingManager.clear();
+inkingManager.importRaw(strokes);
+```
+
+#### Export strokes as an SVG
+
+You can also export your entire drawing within the `InkingManager` to an SVG. The SVG contents will be returned as a string, which you can then store in your server as an `.svg` file.
```
Comment (v-ypalikila) [inline msteams-platform/apps-in-teams-meetings/teams-live-share-canvas.md:352]:
```suggestion
You can export your entire drawing within the `InkingManager` to a scalable vector graphic (SVG). The SVG contents are returned as a string, which you can then store in your server as an .svg file extension.
```

File: msteams-platform/apps-in-teams-meetings/teams-live-share-canvas.md [hunk 2]
```diff
@@ -334,6 +334,28 @@ document.getElementById("line-tip-size").onclick = () => {
 
 You can clear all strokes in the canvas by calling `inkingManager.clear()`. This deletes all strokes from the canvas.
 
+#### Import and export raw strokes
+
+Live Share Canvas supports importing and exporting raw strokes from `InkingManager`, which enables you to export them to your backend for later use in a future session.
```
Comment (v-ypalikila) [inline msteams-platform/apps-in-teams-meetings/teams-live-share-canvas.md:339]:
```suggestion
Live Share Canvas supports importing and exporting raw strokes from `InkingManager`, which enables you to export them to your back-end for later use in a future session.
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-live-share-canvas.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3829f464db903f1eb75a1954e9d4f3efe79e5e5c/msteams-platform%2Fapps-in-teams-meetings%2Fteams-live-share-canvas.md) | 93 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/49300b85-e17d-4cb9-9899-e35f1e47e41e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/teams-live-share-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3829f464db903f1eb75a1954e9d4f3efe79e5e5c/msteams-platform%2Fapps-in-teams-meetings%2Fteams-live-share-capabilities.md) | 93 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f18d1eea-0718-4400-80a3-7e40539c69f2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-live-share-canvas.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fa9f41841fb72125012589c2b797e287f3afeff6/msteams-platform%2Fapps-in-teams-meetings%2Fteams-live-share-canvas.md) | 93 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f47fc000-8fe7-4019-8403-a0ec2e108f7f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/teams-live-share-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fa9f41841fb72125012589c2b797e287f3afeff6/msteams-platform%2Fapps-in-teams-meetings%2Fteams-live-share-capabilities.md) | 93 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0b6c951c-ada6-4b64-bbfb-c77821026d7d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**